pkg/network/node/ovscontroller.go

```diff
@@ -323,64 +329,58 @@ func (oc *ovsController) SetPodBandwidth(hostVeth, sandboxID string, ingressBPS,
 	return nil
 }
 
-func (oc *ovsController) getPodDetailsBySandboxID(sandboxID string) (int, string, string, error) {
+func (oc *ovsController) getPodDetailsBySandboxID(sandboxID string) (int, net.IP, error) {
```

Comment:
So what if we just stuff the PodIP into external-ids?  Then we don't need to dump and parse flows all the time.

Also, what about:

ovs-vsctl --no-heading --data=bare --columns=ofport find interface external-ids:sandbox=XXXXX